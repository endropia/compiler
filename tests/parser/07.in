const a:Integer;
begin end;