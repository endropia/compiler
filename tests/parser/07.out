(1, 16) '=' expected